docs/resources/aviatrix_kubernetes_cluster.md

```diff
@@ -0,0 +1,92 @@
+---
+subcategory: "Secured Networking"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_kubernetes_cluster"
+description: |-
+  Configures a Kubernetes cluster for building Aviatrix Smart Groups from applications running in Kubernetes.
+---
+
+# aviatrix_kubernetes_cluster
+
+The **aviatrix_kubernetes_cluster** resource allows to configure a Kubernetes cluster.
+Aviatrix Smart Groups can be built based on applications running in these clusters .
```

Comment:
```suggestion
The **aviatrix_kubernetes_cluster** resource allows you to configure a Kubernetes cluster. 
Aviatrix Smart Groups can be built based on applications running in these clusters.
```

Reply:
Thank you. I applied the changes locally.